src/kernel/caches.h

```diff
@@ -16,6 +16,9 @@ static constexpr size_t MAX_BLOCK_DB_CACHE{2_MiB};
 //! Max memory allocated to coin DB specific cache (bytes)
 static constexpr size_t MAX_COINS_DB_CACHE{8_MiB};
 
+//! The batch size of DEFAULT_KERNEL_CACHE
+static constexpr size_t DEFAULT_DB_CACHE_BATCH{16_MiB};
+
```

Comment:
In 6fa584fde601dd8fc966fa2496084203c9503215, seems it would make sense to place these two together.

```diff
  //! Suggested default amount of cache reserved for the kernel (bytes)
 static constexpr size_t DEFAULT_KERNEL_CACHE{450_MiB};
+//! Batch size of DEFAULT_KERNEL_CACHE
+static constexpr size_t DEFAULT_DB_CACHE_BATCH{16_MiB};
 //! Max memory allocated to block tree DB specific cache (bytes)
 static constexpr size_t MAX_BLOCK_DB_CACHE{2_MiB};
 //! Max memory allocated to coin DB specific cache (bytes)
 static constexpr size_t MAX_COINS_DB_CACHE{8_MiB};
 
-//! The batch size of DEFAULT_KERNEL_CACHE
-static constexpr size_t DEFAULT_DB_CACHE_BATCH{16_MiB};
-
 namespace kernel {
 struct CacheSizes {
```